File: src/main/java/no/nav/domain/pensjon/regler/beregning2011/BeregningsgrunnlagOrdiner.java
package no.nav.domain.pensjon.regler.beregning2011;

import java.util.ArrayList;
import java.util.Collections;
import java.util.List;

/**
 * @author Steinar Hjellvik (Decisive) - PK-11391
 */
public class BeregningsgrunnlagOrdiner extends AbstraktBeregningsgrunnlag {

    

    private List<OpptjeningUT> opptjeningUTListe;

    private int nasjonaltSnitt;

    /**
     * Angir det sluttpoengtall som ordinært beregningsgrunnlag er omregnet fra.
     * Angår kun beregning av avdøde i sammenheng med nytt UT_GJT.
     */
    private double sluttpoengtall;


}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning2011/BeregningsgrunnlagOrdiner.kt
package no.nav.pensjon.regler.domain.beregning2011

class BeregningsgrunnlagOrdiner : AbstraktBeregningsgrunnlag() {
    var opptjeningUTListe: MutableList<OpptjeningUT> = mutableListOf()
    var nasjonaltSnitt = 0

    /**
     * Angir det sluttpoengtall som ordinårt beregningsgrunnlag er omregnet fra.
     * Angår kun beregning av avdøde i sammenheng med nytt UT_GJT.
     */
    var sluttpoengtall = 0.0
}